docs/src/content/docs/getting-started/secrets-and-variables.mdx

```diff
@@ -57,10 +125,25 @@ workflows:
       - type: apply
 ```
 
-## `.tfvars`
+The following code snippet shows how to set a timestamp for resource tagging:
```

Comment:
This might not be a great example, it depends on what how `deployment_timestamp` is used in the code, but this would cause every resource to get changed on every operation (the timestamp will be different on each run).